docs/fsharp/style-guide/formatting.md

```diff
@@ -832,11 +832,27 @@ with
 
 ## Formatting function parameter application
 
-In general, most function parameter application is done on the same line.
+In general, most function parameter application is done on the same line:
```

Comment:
I'm not really sure this line makes sense - perhaps it's my TypeScript and C# background or unfamiliarity with F#, but the term "function parameter application" to me is just "argument", right?

```suggestion
In general, most arguments are provided on the same line:
```